script.js

```diff
@@ -0,0 +1,61 @@
+let collection = JSON.parse(localStorage.getItem('collection')) || [];
+const add = (title, author) => {
+  const book = { title, author };
+  collection.push(book);
+  localStorage.setItem('collection', JSON.stringify(collection));
+};
+
+const remove = (title) => {
+  collection = collection.filter((book) => book.title !== title);
+  localStorage.setItem('collection', JSON.stringify(collection));
+};
+
```

Comment:
- I think it would be a better idea to use unique ` ID`  property for each book since it would make it easier to remove books with duplicate values